WorkoutTest/ExerciseController.swift

```diff
@@ -167,7 +167,16 @@ class ExerciseController: UIViewController {
   }
   
   private func removeTimerView() {
-    timerView.removeFromSuperview()
+    
+    UIView.animate(withDuration: 0.5, delay: 0.1, animations: {
+      self.nextButton.frame.origin.y -= 100
+      self.backButton.frame.origin.y -= 100
+      self.timerView.alpha = 0
+    }, completion: { (finished) in
```

Comment:
```suggestion
    }, completion: { finished in
```

we dont use a '(' ')' parenthesis in such cases